howto/index.md

```diff
@@ -27,6 +28,7 @@ responsible for managing Charmed HPC deployments.
 :maxdepth: 1
 :hidden:
 
-getting-started/index
+Initialize Cloud Environment <initialize-cloud-environment>
```

Comment:
I'd make "cloud" and "environment" lowercase since they're not proper nouns. E.g. they don't refer to a specific thing, but are a generic way of saying "hey you need to initialize this thing." I look at it like "Azure Cloud" is the proper noun/name while "Initial your cloud" is a generic way of referring to multiple clouds:

```suggestion
Initialize cloud environment <initialize-cloud-environment>
```